main/ertp/api/issuer.md

```diff
@@ -1,214 +1,263 @@
 # Issuer
 
-The issuer cannot mint new amounts, but it can create empty purses and payments. The issuer can also transform payments (splitting payments, combining
-payments, burning payments, and claiming payments exclusively). The issuer should be obtained from a trusted source and then relied upon as the
-authority as to whether an untrusted payment is valid.
+An `issuer` represents the identity of a specific kind of digital asset. 
+While it cannot create new value by creating digital assets like a `mint`, 
+it recognizes and identifies `purses` and `payments` that carry actual value.
+It can create empty `purses` and transform `payments` (by splitting, 
+combining, burning, or exclusively claiming). 
+
+An `issuer` has an unchangeable one-to-one relationship with the `mint` and
+`brand` that were created with it. For any `brand` for which you will accept
+`payments` in, you should obtain its `issuer` from a trusted source.
+You can then rely on that `issuer` as the authority to 
+determine if an untrusted `payment` of that `brand` is valid.
+
+## makeIssuerKit(allegedName, amountMathKind)
+- `allegedName` `{String}` 
+- `amountMathKind` `{MathKind}` - optional
+- Returns `{IssuerKit}`
+
+Makes a new `issuer` as well as its one-to-one associated ERTP objects; a `mint` and a `brand`. 
+All are in unchangeable one-to-one relationships with each other. It also makes a new
+`amountMath`, which is in an unchangeable many-to-one relationship with the new `issuer`.
```

Comment:
```suggestion
All are in unchangeable one-to-one relationships with each other. 
```
I have issues with the many-to-one language, and also I think it's confusing to point out amountMath in the same way as brand if we want the users to make their own amountMath locally.

Reply:
How's this: "While this also returns an `amountMath`, we strongly urge that instead of using that one, which
is asynchronous, you use `makeLocalAmountMath(issuer)` to create a local, synchronous, version
and use that one."